src/components/AboutUs/SubsidiariesSection.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
```

Comment:
I think this is the first time we give the suffix `Section` to a component which represents a section of a page, probably it's best to call the file just `Subsidiaries.js` to comply with the rest?

Reply:
Good point. I thought I had seen this pattern somewhere :thinking: 

Never mind